hexrd/ui/calibration/polarview.py

```diff
@@ -119,6 +119,10 @@ def update_angular_grid(self):
     def angular_grid(self):
         return self._angular_grid
 
+    @property
+    def eta_period(self):
+        return self.eta_min + 360.
```

Comment:
Quick question: should this be a list instead of a single number, like `np.r_[self.eta_min, self.eta_min + 360.]`?

Reply:
Oops, yes, should be array_like exactly how you have suggested.

Reply:
it should be handled exactly how it is in the overlay modules (_e.g._, https://github.com/HEXRD/hexrdgui/blob/01c532b3fe55e69f6ac128268b2479da094b7841/hexrd/ui/overlays/laue_diffraction.py#L101)